### PR TITLE
When a visitor contacts LXD, email the sales team

### PR DESCRIPTION
**Pull request by gabebw**
* A visitor fills out the contact form
* The app sends an email to sales@lxdinc.com

https://trello.com/c/s2G8doJq

### CODE REVIEW

File: app/views/pages/contact_request_thanks.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<h1>
+  <%= t(".thanks") %>
+</h1>
+<%= t(".body_html") %>
```
Comment (gylaz) [inline app/views/pages/contact_request_thanks.html.erb]: Worth wrapping this body message in a `div` or `p`?

File: config/locales/en/pages.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+en:
+  pages:
+    contact_request_thanks:
+      thanks: Thanks
+      body_html: >
```
Comment (gylaz) [inline config/locales/en/pages.yml]: I believe `>` doesn't actually have any effect in Rails i18n YAML.

File: spec/features/contact_spec.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+feature "User contacts LXD" do
+  scenario "and sees a thank you message" do
+    visit root_path
+    click_on t("application.nav.contact_us")
+
+    fill_out_form
+    click_on "Submit"
+
+    expect(page).to have_content(t("pages.contact_request_thanks.thanks"))
```
Comment (gylaz) [inline spec/features/contact_spec.rb]: `have_content` -> `have_text`?

File: app/views/pages/contact_request_thanks.html.erb (hunk 1 above)
Comment (gabebw) [inline app/views/pages/contact_request_thanks.html.erb]: Usually I'd agree, but this is intentionally almost completely unstyled while I wait for Tyler to come in afterwards and make it look good.

File: config/locales/en/pages.yml (hunk 2 above)
Comment (gabebw) [inline config/locales/en/pages.yml]: Per the [official docs](http://symfony.com/doc/current/components/yaml/yaml_format.html):

> strings can be written with the folded style, denoted by >, where each line break is replaced by a space

I think I'd prefer this to render spread over multiple lines, though, so I'm going to remove it.

File: spec/features/contact_spec.rb (hunk 3 above)
Comment (gabebw) [inline spec/features/contact_spec.rb]: Good call.

**Comment by gabebw** [comment]
@gylaz I think I've addressed all of your feedback.

**Comment by gabebw** [comment]
@gylaz Oh, I see now that you approved it.

File: config/locales/en/pages.yml (hunk 2 above)
Comment (gylaz) [inline config/locales/en/pages.yml]: TIL

**Pull request merged**